contracts/SplitEquippable/README.md

```diff
@@ -709,274 +709,253 @@ async function mintTokens(
+      equippableRefIdLeftGem,
+      baseAddress,
+      `ipfs://gems/typeA/left.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
-      // Equipped into mid slot for first type of gemNesting
-      {
-        id: 3,
-        equippableGroupId: equippableRefIdMidGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeA/mid.svg`,
-      },
+    await gem.addAssetEntry(
+      // Equipped into mid slot for first type of gem
+      equippableRefIdMidGem,
+      baseAddress,
+      `ipfs://gems/typeA/mid.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
-      // Equipped into left slot for first type of gemNesting
-      {
-        id: 4,
-        equippableGroupId: equippableRefIdRightGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeA/right.svg`,
-      },
+    await gem.addAssetEntry(
+      // Equipped into left slot for first type of gem
+      equippableRefIdRightGem,
+      baseAddress,
+      `ipfs://gems/typeA/right.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
-      // Full version for second type of gemNesting, no need of refId or base
-      {
-        id: 5,
-        equippableGroupId: 0,
-        baseAddress: ethers.constants.AddressZero,
-        metadataURI: `ipfs://gems/typeB/full.svg`,
-      },
+    await gem.addAssetEntry(
+      // Full version for second type of gem, no need of refId or base
+      0,
+      ethers.constants.AddressZero,
+      `ipfs://gems/typeB/full.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
-      // Equipped into left slot for second type of gemNesting
-      {
-        id: 6,
-        equippableGroupId: equippableRefIdLeftGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeB/left.svg`,
-      },
+    await gem.addAssetEntry(
+      // Equipped into left slot for second type of gem
+      equippableRefIdLeftGem,
+      baseAddress,
+      `ipfs://gems/typeB/left.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
-      // Equipped into mid slot for second type of gemNesting
-      {
-        id: 7,
-        equippableGroupId: equippableRefIdMidGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeB/mid.svg`,
-      },
+    await gem.addAssetEntry(
+      // Equipped into mid slot for second type of gem
+      equippableRefIdMidGem,
+      baseAddress,
+      `ipfs://gems/typeB/mid.svg`,
       [],
       []
     ),
-    await gem.addResourceEntry(
-      // Equipped into right slot for second type of gemNesting
-      {
-        id: 8,
-        equippableGroupId: equippableRefIdRightGem,
-        baseAddress: baseAddress,
-        metadataURI: `ipfs://gems/typeB/right.svg`,
-      },
+    await gem.addAssetEntry(
+      // Equipped into right slot for second type of gem
+      equippableRefIdRightGem,
+      baseAddress,
+      `ipfs://gems/typeB/right.svg`,
```

Comment:
I'd add the keys to all of these back (it's a bit much to add suggestions to all of them in the GUI of GitHub PR reviewer) 😂